deeppavlov/models/bert/bert_classifier.py

```diff
@@ -0,0 +1,253 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from logging import getLogger
+from typing import List, Dict, Union
+
+import tensorflow as tf
+from bert_dp.modeling import BertConfig, BertModel
+from bert_dp.optimization import AdamWeightDecayOptimizer
+from bert_dp.preprocessing import InputFeatures
+
+from deeppavlov.core.commands.utils import expand_path
+from deeppavlov.core.common.registry import register
+from deeppavlov.core.models.lr_scheduled_tf_model import LRScheduledTFModel
+
+logger = getLogger(__name__)
+
+
+@register('bert_classifier')
+class BertClassifierModel(LRScheduledTFModel):
+    """Bert-based model for text classification.
+
+    It uses output from [CLS] token and predicts labels using linear transformation.
+
+    Args:
+        bert_config_file: path to Bert configuration file
+        n_classes: number of classes
+        keep_prob: dropout keep_prob for non-Bert layers
+        one_hot_labels: set True if one-hot encoding for labels is used
+        multilabel: set True if it is multi-label classification
+        return_probas: set True if return class probabilites instead of most probable label needed
+        attention_probs_keep_prob: keep_prob for Bert self-attention layers
+        hidden_keep_prob: keep_prob for Bert hidden layers
+        optimizer: name of tf.train.* optimizer or None for `AdamWeightDecayOptimizer`
+        num_warmup_steps:
+        weight_decay_rate: L2 weight decay for `AdamWeightDecayOptimizer`
+        pretrained_bert: pretrained Bert checkpoint
+        min_learning_rate: min value of learning rate if learning rate decay is used
+    """
+    # TODO: add warmup
+    # TODO: add head-only pre-training
+    def __init__(self, bert_config_file, n_classes, keep_prob,
+                 one_hot_labels=False, multilabel=False, return_probas=False,
+                 attention_probs_keep_prob=None, hidden_keep_prob=None,
+                 optimizer=None, num_warmup_steps=None, weight_decay_rate=0.01,
+                 pretrained_bert=None, min_learning_rate=1e-06, **kwargs) -> None:
+        super().__init__(**kwargs)
+
+        self.return_probas = return_probas
+        self.n_classes = n_classes
+        self.min_learning_rate = min_learning_rate
+        self.keep_prob = keep_prob
+        self.one_hot_labels = one_hot_labels
+        self.multilabel = multilabel
+        self.optimizer = optimizer
+        self.num_warmup_steps = num_warmup_steps
```

Comment:
`num_warmup_steps` is unused. It is possible to pass it to `get_train_op` via `kwargs` and parse it inside, but may be it is unnecessary.

Reply:
it will be used in future releases, we have TODO for this